src/calculateRentalCost.js

```diff
@@ -1,10 +1,12 @@
-/**
- * @param {number} days
- *
- * @return {number}
- */
 function calculateRentalCost(days) {
-  // write code here
-}
+  let totalCost = days * 40;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
+    totalCost -= 20;
```

Comment:
You can simplify the code by removing the `else` keyword since the `if` block contains a `return` statement.